test/fit_tests/common/test_api_utils.py

```diff
@@ -0,0 +1,521 @@
+# Copyright 2016, EMC, Inc.
+
+"""
+ RackHD 1.1 common test utlitiies
+
+ authors: ehohenstein, yuchi zhang
+
+ Purpose/intent
+ This utility library contain helper functions for parsing response data from the RackHD APIs.
+"""
+
+import sys
+import subprocess
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/common")
+import fit_common
+
+import unicodedata
+import json
+
+##########################################
+#  Node List Utils
+##########################################
+
+def rackhd_get_node_list():
```

Comment:
@jlongever had designed a feature that auto-generates the RackHD API library from RackHD swagger file, so we don't need to create a lot util functions for each API call, and also that library can be auto-synced even if there is some API change. see https://github.com/jlongever/on-http-api2.0-java

Are you aware of this feature? I think this feature this is pretty good.